python/tank/commands/setup_project_params.py

```diff
@@ -578,7 +582,8 @@ def preview_project_path(self, storage_name, project_name, platform):
         # append the project name
         storage_path += "/%s" % project_name
         # note that project name can be 'foo/bar' with a forward slash for all platforms
-        if is_windows():
+        # Fixes any slashes that might be in the wrong direction.
+        if platform == "win32":
```

Comment:
This was a regression introduced by the branch. The caller is passing in a platform name for every single platform it wants a preview for, so the current platform is not relevant. Annoyingly, the API expects the use of sys.platform compatible values, but it's too late to fix that, as the admin ui will pass in those values.